### DISCUSSION TITLE
API call to check if someone is actively using Stash (e.g. watching a video)

### DISCUSSION THREAD

**Discussion by BF708C50B654** [stashapp/stash#5551] [Q&A] [answered]
Hi. 
I am fairly new to GraphQL. Is there a way to check, if someone is actively using Stash (e.g. watching a video or viewing photos)? 

My home server is locally hosting Plex and Stash. I have a Home Assitant automation to shut down the server after one hour of Plex inactivity. I would like to add additional check for Stash.

**Comment by feederbox826** [comment, marked as answer]
There is no API and play history would be hard to track, your best bet is looking at access logs from a reverse proxy
